tests/unit/IncorporationDateTime.spec.ts

```diff
@@ -0,0 +1,128 @@
+// Libraries
+import Vue from 'vue'
+import Vuelidate from 'vuelidate'
+import Vuetify from 'vuetify'
+import { getVuexStore } from '@/store'
+
+// Utils
+import { createLocalVue, mount } from '@vue/test-utils'
+
+// Components
+import { IncorporationDateTime } from '@/components/ReviewConfirm'
+
+Vue.use(Vuetify)
+Vue.use(Vuelidate)
+const vuetify = new Vuetify({})
+
+// Store
+const store = getVuexStore()
+document.body.setAttribute('data-app', 'true')
+
+describe('List Shares and Series', () => {
+  let wrapperFactory: any
+
+  const dateTimeDefault = {
+    valid: false,
+    isImmediate: false,
+    isFutureEffective: false,
+    futureEffectiveDate: null
+  }
+
+  beforeEach(() => {
+    const localVue = createLocalVue()
+
+    wrapperFactory = (propsData) => {
+      return mount(IncorporationDateTime, {
+        propsData: {
+          ...propsData
+        },
+        localVue,
+        store,
+        vuetify
+      })
+    }
+  })
+
+  afterEach(async () => {
+  })
+
+  it('confirms no default Date Time Selection', () => {
+    const wrapper = wrapperFactory({ incorporationDateTime: dateTimeDefault })
+
+    // Reference the Radios
+    const radioInput = wrapper.findAll('input[type="radio"]')
+    const radioIsImmediate = radioInput.at(0)
+    const radioIsFutureEffective = radioInput.at(1)
+
+    // Verify radios are false
+    expect(radioIsImmediate.attributes('aria-checked')).toBe('false')
+    expect(radioIsFutureEffective.attributes('aria-checked')).toBe('false')
+  })
+
+  it('confirms the selector fields are disabled if future effective is NOT selected', () => {
```

Comment:
I'd like to see the Immediate radio button clicked in this test (opposite of test below).

Reply:
Good call, can do. Will update this